backend/dataall/base/cdkproxy/README.md

```diff
@@ -26,11 +26,12 @@ from aws_cdk import (
 )
 from dataall.base.cdkproxy.stacks import stack
 
+
```

Comment:
this is also not refering to the StackRepository but to aws_cdk.Stack.

The whole block should be:

```

from aws_cdk import (
    aws_s3 as s3,
    aws_sqs as sqs,
    core,
    Stack
)


@stack(stack="mypredefinedstack")
class MyPredefinedStack(Stack):
    def __init__(self, scope, id, **kwargs):
        super().__init__(scope, id, **kwargs)
        # constructs goes here

```